src/app/index.tsx

```diff
@@ -24,15 +24,52 @@ import GOVERNANCE_ABI from "libs/abis/vote-governance.json";
 import SNOWCONE_ABI from "libs/abis/snowcone.json";
 import FEE_DISTRIBUTOR_ABI from "libs/abis/fee-distributor.json";
 import { CONTRACTS } from "config";
-import { StakingPage } from "./pages/StakingPage";
+import { StakingPage } from "./pages/Staking";
 import Layout from "./Layout";
-import { LiquidityPage } from "./pages/Liquidity/Loadable";
-import { RewardsPage } from "./pages/Rewards/Loadable";
 import { SwapPage } from "./pages/Swap/Loadable";
 import { RisksPage } from "./pages/Risks/Loadable";
+import { LiquidityPage } from "./pages/Liquidity/Loadable";
+import { RewardsPage } from "./pages/Rewards/Loadable";
+import { Swap } from "./containers/Swap";
+import SWAP_ROUTER_ABI from "abi/swapRouter.json";
+import { tokens } from "./tokens";
+import { Token, TokenSymbols } from "./containers/Swap/types";
+import { Rewards } from "./containers/Rewards";
+import { pools } from "./pools";
+import { GlobalActions, useGlobalSlice } from "store/slice";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Web3Selectors } from "./containers/BlockChain/Web3/selectors";
+
+// import { Swap } from "./containers/Swap";
```

Comment:
remove unnecessary commented out code or please add a description why is commented out.